TiledLib/Map.cs

```diff
@@ -38,6 +38,9 @@ public class Map : IXmlSerializable
         [JsonProperty("tileheight")]
         public int CellHeight { get; set; }
 
+        [JsonProperty("hexsidelength")]
+        public int? HexSideLength { get; set; }
```

Comment:
I chose to make these new elements nullable as it seems that Tiled doesn't generate them if they are not relevant.